soda/core/tests/data_source/test_anomaly_check.py

```diff
@@ -43,20 +44,29 @@ def test_anomaly_detection_not_have_enough_data(scanner: Scanner):
     )
 
     scan.execute(allow_warnings_only=True)
+    scan_cloud_result = mock_soda_cloud.pop_scan_result()
+    assert scan_cloud_result["checks"][0]["outcomeReasons"] == [
+        {
+            "code": "notEnoughHistory",
+            "message": "Skipping anomaly metric check eval because there is not enough historic data yet",
+            "severity": "warn",
+        }
+    ]
 
 
+# TODO: Ask Baturay why we test this?
```

Comment:
@bastienboutonnet  The idea was to test anomaly detection for cases having no previous data. "not enough data" and "no data" are handled at different locations at soda-core, for this reason I added two different tests.